### DISCUSSION TITLE
Error when running make_session.py

### DISCUSSION THREAD

**Discussion by joshegreenfield2** [iluvcapra/py-ptsl#46] [Q&A]
Hi,

I keep getting this error message when trying to run make_session.py

Exception has occurred: CommandError
ErrType 100: PT_UnknownError (is_cloud_project)
  File "/Users/py-ptsl/ptsl/client.py", line 161, in run
    raise CommandError(command_errors.errors)
  File "/Users/.../py-ptsl/ptsl/builders/create_session_builder.py", line 75, in create
    self._engine.client.run(op)
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^
  File "/Users/.../py-ptsl/examples/make_session.py", line 24, in <module>
    builder.create()
    ~~~~~~~~~~~~~~^^
ptsl.errors.CommandError: ErrType 100: PT_UnknownError (is_cloud_project)

I have been able to successfully run GET functions, like print_tracks.py and session_info.py but I cant do any POST commands like make_session.py. Please advise. I don't see any refernce to cloud projects in the python code, but I do see on the Avid documentation

Thank you!

**Comment by iluvcapra** [comment]
Hi Josh, thanks for submitting the ticket! Can you please post what version of Pro Tools you are using?

**Reply by joshegreenfield2** [reply]
Hey I’m using ProTools Ultimate 2024.10.0

**Reply by iluvcapra** [reply]
I'll have to take a look when I have some time, are you using the example code literally or have you modified it at all?

**Reply by joshegreenfield2** [reply]
im using the example script provided make_session.py and have not modified it at all

**Reply by oudileoudi** [reply]
hello 
same problem here.
and it's maybe the reason why create session failed (iCloud session) tthe True false prob

protools 2024.10.0

**Comment by iluvcapra** [comment]
I've made a small change in client.py that fixes this, it's presently on master can you give it a try and let me know if it works for you?

**Reply by oudileoudi** [reply]
thank you it 's working
by th e way it's possible to add 1 track with some elements?

**Reply by joshegreenfield2** [reply]
Working for me too. Thank you!!
